src/lineup/desc.ts

```diff
@@ -7,7 +7,7 @@ import {extent} from 'd3';
 import {IAnyVector} from 'phovea_core/src/vector';
 import {VALUE_TYPE_STRING, VALUE_TYPE_CATEGORICAL, VALUE_TYPE_REAL, VALUE_TYPE_INT} from 'phovea_core/src/datatype';
 
-export function numberCol(col: string, rows: any[], label = col, visible = true, width = -1, selectedId = -1) {
+export function numberCol(col: string, rows: any[], label = col, visible = true, width = -1, selectedId = -1, selectionOptions = undefined) {
```

Comment:
can the selectedOptions be more specified? it is supposed to be an object? string?, number?